cmd/kyma/alpha/list/module/opts.go

```
@@ -88,7 +88,10 @@ func (o *Options) validateChannel() error {
 			ChannelMinLength, ChannelMaxLength, ErrChannelValidation,
 		)
 	}
-	matched, _ := regexp.MatchString(`^[a-z]+$`, o.Channel)
+	matched, err := regexp.MatchString(`^[a-z]+$`, o.Channel)
+	if err != nil {
+		return nil
```

Please return the err here